sklearn/covariance/graph_lasso_.py

```diff
@@ -21,7 +21,8 @@
 from ..utils.validation import check_random_state, check_array
 from ..linear_model import lars_path
 from ..linear_model import cd_fast
-from ..cross_validation import _check_cv as check_cv, cross_val_score
+from ..model_selection.partition import _check_cv as check_cv
+from ..model_selection.validation import cross_val_score
```

Comment:
> internal use

Sorry, I was a bit ambiguous. I don't think we get any benefit from using the longer import path in the codebase. If anything, we get a drawback: if we end up renaming, say, `split` to `partition`, we'd need to change many files.

What I meant by internal use is private (underscored) functions such as `_check_cv`. Not sure what to do about that.
Also I'd like to hear whether my understanding of this aligns with others; that the division into {validation, search...} should be completely transparent to the user.  I think we should avoid providing two equivalent public import paths for the same thing. This could be solved by calling the submodules _validation, _search, etc.

Ping @amueller, @GaelVaroquaux maybe?